content/posts/newsletter/2019-09-03.md

```diff
@@ -0,0 +1,188 @@
+---
+title: The XMPP Newsletter, 03 September 2019
+save_as: 2019/09/newsletter-03-september/index.html
+url: 2019/09/newsletter-03-september/
+date: 2019-09-03 09:00
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the September edition of the XMPP Newsletter!
+
+Loads of news for these two months of July and August: two events in Lyon and Stockholm (more to come), new tools, new server releases (Openfire, ejabberd), lots of client releases (Kaidan, Salut à Toi, Xabber Android, Movim, Converse, Beagle IM and Siskin IM). We also experiment a new section about XEP, or specifications. The XMPP/Jabber community is alive and kicking!
```

Comment:
> The XMPP/Jabber community is alive and kicking!

I'd remove this line. By leaving it in, the suggestion is created that we need to tell this to people. :)

Reply:
Done, thanks!